examples/official-storybook/webpack.config.js

```diff
@@ -5,7 +5,39 @@ module.exports = async ({ config }) => ({
   module: {
     ...config.module,
     rules: [
-      ...config.module.rules,
+      ...config.module.rules.slice(1),
+      {
+        test: /\.(mjs|jsx?|tsx?)$/,
+        use: [
+          {
+            loader: 'babel-loader',
+            options: {
+              cacheDirectory: `.cache/storybook`,
+              presets: [
+                ['@babel/preset-env', { shippedProposals: true, useBuiltIns: 'usage', corejs: 2 }],
+                '@babel/preset-typescript',
+                ['babel-preset-minify', { builtIns: false, mangle: false }],
```

Comment:
@ndelangen shouldn't we apply this one only in prod mode?

Reply:
yeah